src/compare_covermaps.py

```diff
@@ -582,7 +624,7 @@ def generate_report(dataset_name: str, country: str, true, pred) -> pd.DataFrame
                 ee.Image("users/eutzschn/Ethiopia_Bure_Jimma_2020_v1"),
                 ee.Image("users/izvonkov/Ethiopia_Bure_Jimma_2019_v1"),
             ]
-        ),
+        )""",
```

Comment:
Decided to push the earthengine collections as strings so they can be executed on GEE in Javascript easily

Reply:
LGTM. I like the P/R/F1 plot and sorting by F1 score is helpful.